build/pipelines/release.yml

```diff
@@ -65,11 +68,11 @@ jobs:
             BuildConfiguration: ${{ config }}
             BuildPlatform: ${{ platform }}
   displayName: Build
+  timeoutInMinutes: 240
   cancelTimeoutInMinutes: 1
   steps:
   - checkout: self
     clean: true
-    fetchDepth: 1

```

Comment:
PGO can't resolve from peer branches with depth 1 here, so I'll allow it for release builds.